docs/blog/newsletter-september-2022.rst

```diff
@@ -0,0 +1,91 @@
+.. post:: September 06, 2022
+   :tags: newsletter
+
+##########################################
+Write the Docs Newsletter – September 2022
+##########################################
+
+Hey, everyone, the newsletter is back! We took one of our regularly scheduled breaks, but now we're back with regularly exceptional insights from our community.
+
+If you'd like to get more of those insights, it's your last chance to get `tickets for the Prague conference <https://www.writethedocs.org/conf/prague/2022/tickets/>`__. Ticket sales end on September 8. Then the conference starts September 11. We'll certainly all have lots of new ideas to think about afterwards.
+
+And if you have insights of your own to share, there's still (a little) time to sign up to present at the Australia conference. For more details, see the `Call for Proposals <https://www.writethedocs.org/conf/australia/2022/cfp/>`__. The deadline for submissions is September 16. And you can join the conference to see all the amazing ideas on December 8--9.
```

Comment:
Triple use of insights here. Thoughts? Opinions?

You're not signing up to present, you're proposing. It's a nit, but a meaningful one imo

Reply:
It was quadruple overall, which :see_no_evil:

Reply:
And yes, very meaningful and embarrassing for me to have left it that way.